cli/azd/CHANGELOG.md

```diff
@@ -1,15 +1,37 @@
 # Release History
 
-## 1.7.0-beta.1 (Unreleased)
+## 1.7.0 (2024-03-12)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
+- [[3450]](https://github.com/Azure/azure-dev/pull/3450) Adds support for pushing container images to external container registries
+- [[3216]](https://github.com/Azure/azure-dev/pull/3216) Adds support for OneAuth on windows
```

Comment:
Remove OneAuth support Marina. It's still under development. Will eventually be rolled out later.